eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -43,9 +43,14 @@ jobs:
       container: $[ variables['Container'] ]
 
     steps:
-      - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
+      - template: /eng/pipelines/templates/steps/initialize-test-environment.yml
         parameters:
-          AgentImage: $(OSVmImage)
+          Artifacts: ${{ parameters.Artifacts }}
+          AdditionalModules: ${{ parameters.AdditionalModules }}
+          Paths:
+            - '${{ parameters.ServiceDirectory }}'
```

Comment:
How do we verify this doesn't break anything? I assume we need to run all test pipelines to truly verify. If we find a specific test pipeline that needs additional files how do they fix that? Should we add another template parameter that we pass through from live.tsts.yml?

Reply:
It's a little harder even than running all pipelines, because I worry there could be cases where we don't include certain files and tests still pass (since they'll run against found/built files). So I think the approach of a large cross-section for general pass/fail/duration parity + a small cross-section that I go over with a fine-toothed comb for differences (or script to compare test counts in the logs), is the way to go here.

I don't anticipate finding places where we require additional files, but if we do, I'd rather handle it outside of the pipeline config (i.e. via the scripts or some sort of external config), to lay the groundwork for more explicit configuration of a broad set of test file dependencies (similar to the problem we have with non-trivial test-resources.json use cases).